src/bin/daemon.rs

```diff
@@ -19,7 +19,10 @@ fn parse_args(args: Vec<String>) -> Option<PathBuf> {
         return None;
     }
 
-    if args[1] != "--conf" {
+    if args[1] == "--version" || args[1] == "-v" {
+        eprintln!("{}", VERSION);
+        process::exit(1);
+    } else if args[1] != "--conf" {
         eprintln!("Only a single command line argument is supported: --conf. All other configuration parameters must be specified in the configuration file.");
```

Comment:
Would it be worth updating this message? Maybe not as this message is more about running the daemon.

Reply:
not sure it worth it, @darosior?

Reply:
Agree with @jp1ac4, but let's keep that for future work.